public/app/features/trails/Breakdown/LabelBreakdownScene.tsx

```diff
@@ -301,6 +301,7 @@ export function buildAllLayout(
     const unit = queryDef.unit;
 
     const vizPanel = PanelBuilders.timeseries()
+      .setOption('legend', { showLegend: false })
```

Comment:
This won't fix the issue, as https://github.com/grafana/grafana/pull/94284/files#diff-3c8bff0c7802ec237d92e6dc9089b57259fc59a33ba3ca813867826ba21c0984L324 is still being called. Also, I think you want these two options:

```
  .setOption('tooltip', { mode: TooltipDisplayMode.Multi, sort: SortOrder.Descending })
  .setOption('legend', { showLegend: false })
```

Reply:
I set options on all PanelBuilders.timeseries() so that the any of the activation handlers (there are two, as you pointed out) have options to change on activation. If we have already added the options and they are the same for all timeseries panels we are building, do we even need the activation handlers anymore? cc @darrenjaneczek 

```
    vizPanel.addActivationHandler(() => {
      vizPanel.onOptionsChange(breakdownPanelOptions);
    });
```

Reply:
I just commented out those parts of the code on lines 325 and 388
```
    // vizPanel.addActivationHandler(() => {
    //   vizPanel.onOptionsChange(breakdownPanelOptions);
    // });
```

and everything still appears to be working 🤞🏻 but I am very new to the panel building. Does logs use addActivationHandlers? Am I right in thinking we can just remove these if the panel options are always the same?

Reply:
I added the other options, resolving this issue, thank you!